junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/extension/TimeoutExtension.java

```diff
@@ -10,6 +10,9 @@
 
 package org.junit.jupiter.engine.extension;
 
+import static org.junit.jupiter.engine.config.JupiterConfiguration.TIMEOUT_MODE_PROPERTY_NAME;
+
+import java.lang.management.ManagementFactory;
```

Comment:
Please add `requires java.management;` to `junit-jupiter-engine`'s [module-info.java](https://github.com/junit-team/junit5/blob/master/junit-jupiter-engine/src/module/org.junit.jupiter.engine/module-info.java#L12) file and update the expected list of requirements in this integration test: [junit-jupiter-engine.expected.txt](https://github.com/junit-team/junit5/blob/master/platform-tooling-support-tests/projects/jar-describe-module/junit-jupiter-engine.expected.txt)